main.py

```diff
@@ -11,27 +16,44 @@
 # register REST endpoints
 # task #2 + #6
 
+
+@app.get("/players/", response_model=List[schemas.PlayerBase])
+def read_users(db: Session = Depends(get_db)):
+    users = get_users(db)
```

Comment:
można w sumie wywalić tą funkcję `get_users` skoro to jest tylko jedna linijka i po prostu wrzucić jej zawartość tutaj